docs/usage.md

```diff
@@ -106,6 +107,23 @@ it was previously clean. If any scripting depends on the cleanliness of a git re
 the changes, the scripting should either be changed to handle the dirty status or the changes should be temporarily
 stashed by wrapping in `git stash && <command> && git stash pop` according to the suitability of the context.*
 
+### Merge SBOMs
+
+Sometimes it might be necessary to merge two or more SBOMs. This could be done with `cachi2 merge-sboms`:
+
+```shell
+cachi2 merge-sboms <cachi2_sbom_1.json> ... <cachi2_sbom_n.json>
+```
+
+The subcommand expects at least two SBOMs, all in Cachi2 format and would exit with error if any of the SBOMs is in
```

Comment:
I wonder if calling it `Cachi2 format` would be misleading. We're pretty much following the official CycloneDX specification, with some custom properties (the ones prefixed with `cachi2:`).

Maybe we could phrase this as:

> The subcommand expects at least two SBOMs, all produced by Cachi2 and will exit with error otherwise. The reason for this is that Cachi2 supports a [limited set](https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/core/models/sbom.py#L7-L13) of component [properties](https://cyclonedx.org/docs/1.4/json/#components_items_properties), and it validates that no other properties exist in the SBOM.

Reply:
Done